### PR TITLE
fixed script to user iam role policy attachment

### PR DESCRIPTION
**Pull request by dhavalocked**

### CODE REVIEW

File: terraform/environments/fetureflag/main.tf [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# ----------------------------------------------------------------------------------------------------------------------
```
Comment (VivekFitkariwala) [inline terraform/environments/fetureflag/main.tf]: We don't need this in the commit....lets keep it local

File: terraform/environments/fetureflag/tfvars.sample [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+autoscaling_max_size    =  4 
+
+# APP environment variables
+GOOGLE_CLIENT_ID        = "google-project-id"
+MONITOR_PASSWORD        = "password"
+SWAGGER_PASSWORD        = "password"
+
+# CICD variables
+repository_name         = "educational-experiment-service"
+branch_name             = "fetureflag"
```
Comment (VivekFitkariwala) [inline terraform/environments/fetureflag/tfvars.sample]: fetureflag -> featureflag

File: terraform/environments/fetureflag/main.tf (hunk 1 above)
Comment (dhavalocked) [reply terraform/environments/fetureflag/main.tf]: It will create problems if CL people creates multiple envs

Comment (dhavalocked) [reply terraform/environments/fetureflag/main.tf]: you can merge now

**Pull request merged**